registry.json

```diff
@@ -2505,5 +2505,20 @@
     "telegram":"jbradach",
     "operatorName":"James Bradach",
     "github":"SolidarityFinance"
-  }
+  },
+  {
+   "ticker":"PLOT",
+   "poolId":"45e00d4c106ecc6c84b21a77f7cb52b1634e5c1e697f4ebfebfff2cf",
+   "social":{
+     "facebook":"https://www.facebook.com/Plot-Cardano-Stake-pool-103749598506160",
+     "twitter":"https://twitter.com/CardanoPlot",
+     "website":"https://www.plotcardano.com",
+     "reddit":"",
+     "instagram":"https://www.instagram.com/plotcardano/",
+     "youtube":"https://www.youtube.com/channel/UCMJHRgaMgUwnxjRIYmk4kfA"
+   },
+   "telegram":"plotcardano",
```

Comment:
We need a personal telegram handle, this is for voting purposes.

Reply:
Updated with user name.